src/conmon.c

```diff
@@ -89,6 +89,11 @@ int main(int argc, char *argv[])
 	/* before we fork, ensure our children will be reaped */
 	atexit(reap_children);
 
+	/* If we were passed a sd-notify socket to use, set it up now */
+	if (opt_sdnotify_socket != NULL) {
```

Comment:
can we just do `if (opt_sdnotify_socket)`

Reply:
AYW